### PR TITLE
Edited Catalog Price Rule page style

### PR DESCRIPTION
**Pull request by DmitryTsymbal**
## Purpose of this pull request

Edited Catalog Price Rule page style

## Affected documentation pages

https://docs.magento.com/user-guide/marketing/price-rules-catalog.html

### CODE REVIEW

**Comment by dobooth** [comment]
@DmitryTsymbal linting please

**Comment by DmitryTsymbal** [comment]
@dobooth , thank you for your attentiveness. Done ✅

File: src/marketing/price-rules-catalog.md [hunk 1]
```diff
@@ -10,10 +10,24 @@ For example, you can define and set the conditions for a price rule that when me
 
 - {:.ce-only}For recurring promotions in Magento Open Source, you can manually set a saved rule to Active or Inactive status each time you want to run the promotion.
 
+## Access catalog price rules
+
+1. On the _Admin_ sidebar, go to **Marketing** > _Promotions_ > **Catalog Price Rules**.
+
 ![Catalog price rules]({% link marketing/assets/price-rule-catalog-grid-ee.png %}){: .zoom}
```
Comment (leslietilling) [inline src/marketing/price-rules-catalog.md:17]: This is not indented and will break numbering

Comment (DmitryTsymbal) [reply src/marketing/price-rules-catalog.md]: Thank you @leslietilling .
Done

**Review by leslietilling — APPROVED** [review]

**Comment by leslietilling** [comment]
@magento import pr to magento-devdocs/merchdocs

**Comment by m2-github-services** [comment]
@leslietilling an error occurred during the Pull Request import.

**Comment by DmitryTsymbal** [comment]
Closing this one and opening https://github.com/magento/merchdocs/pull/1694
Thanks!

**Pull request closed without merging**